spec/have_tag_spec.rb

```diff
@@ -669,4 +669,16 @@
       end
     end
   end
+
+  context 'html and body elements' do
+    asset 'document'
+
+    it 'should find the html element' do
+      expect(rendered).to have_tag('html')
+    end
+
+    it 'should find the body element' do
+      expect(rendered).to have_tag('body')
+    end
```

Comment:
suggest adding examples for `.to_not have_tag(...)`, without them it may be easy to introduce same bug in future again.